yarn.lock

```diff
@@ -4063,11 +4072,16 @@ md5@^2.3.0:
     crypt "0.0.2"
     is-buffer "~1.1.6"
 
-media-chrome@^3.2.5, media-chrome@~3.2.5:
+media-chrome@^3.2.5:
   version "3.2.5"
   resolved "https://registry.yarnpkg.com/media-chrome/-/media-chrome-3.2.5.tgz#657722a216e3c0f877d9522fce7425e25da7e137"
   integrity sha512-tTsgS7x77Bn4p/wca/Si/7A+Q3z9DzKq0SOkroQvrNMXBVyQasMayDcsKg5Ur5NGsymZfttnJi7tXvVr/tPj8g==
 
+media-chrome@^4.1.0, media-chrome@~4.1.1:
+  version "4.1.3"
```

Comment:
using latest version 👍 

you're right though about player.style and media-chrome dep. 
has the potential to conflict